--- deps/orc-1.6.6/cmake_modules/ThirdpartyToolchain.cmake	2020-12-03 21:46:53.000000000 +0100
+++ deps/updated/ThirdpartyToolchain.cmake	2021-01-02 21:15:31.912435700 +0100
@@ -228,8 +228,8 @@
 
 if (WIN32)
   ExternalProject_Add(tzdata_ep
-    URL "ftp://cygwin.osuosl.org/pub/cygwin/noarch/release/tzdata/tzdata-2020b-1.tar.xz"
-    URL_HASH MD5=1BE1D18B4042A5011E96D20054BEEF33
+    URL "ftp://cygwin.osuosl.org/pub/cygwin/noarch/release/tzdata/tzdata-2020c-1.tar.xz"
+    URL_HASH MD5=F2D518459897B8AAAF9221F3F17AD410
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND "")
@@ -300,7 +300,7 @@
                           -Dprotobuf_BUILD_TESTS=OFF)
   if (MSVC)
     set(PROTOBUF_STATIC_LIB_PREFIX lib)
-    list(APPEND PROTOBUF_CMAKE_ARGS -Dprotobuf_MSVC_STATIC_RUNTIME=OFF
+    list(APPEND PROTOBUF_CMAKE_ARGS -Dprotobuf_MSVC_STATIC_RUNTIME=ON
                                     -Dprotobuf_DEBUG_POSTFIX=)
   else ()
     set(PROTOBUF_STATIC_LIB_PREFIX ${CMAKE_STATIC_LIBRARY_PREFIX})
